packages/fuels-contract/src/contract.rs

```diff
@@ -30,6 +29,10 @@ pub struct CompiledContract {
     pub salt: Salt,
 }
 
+// This constant is used to determine the amount in the 1 UTXO
+// when initializing wallets for now.
+pub const DEFAULT_COIN_AMOUNT: u64 = 1_000_000;
```

Comment:
I liked it better in `constants`, any reason you need it here?

Reply:
why is it public if you don't want to leak it 🤔

Reply:
Renamed it and returned it to constans